src/sentry/api/endpoints/organization_events_measurements_histogram.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import
+
+import sentry_sdk
+
+from sentry import features
+from rest_framework import serializers
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEventsV2EndpointBase, NoProjects
+from sentry.snuba import discover
+
+# The maximum number of measurements allowed to be queried at at time
+MAX_MEASUREMENTS = 4
+
+
+class MeasurementsHistogramSerializer(serializers.Serializer):
+    num_buckets = serializers.IntegerField(min_value=1)
+    precision = serializers.IntegerField(default=0, min_value=0, max_value=4)
```

Comment:
Is there a possibility we can default this to 4?

```suggestion
    precision = serializers.IntegerField(default=4, min_value=0, max_value=4)
```

Reply:
is there a reason why you'd want 4 digits of precision here? For web vitals this means we're looking at microseconds rather than milliseconds.

Reply:
Hm.... I can see this being useful when dealing with FID but if that's the reason, I'd lean towards having the front end control this parameter by increasing the precision when the min/max is set and we know the range is small.

Reply:
Will be leaving the precision default to 0 and setting this on the frontend as needed.